daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md

```diff
@@ -0,0 +1,421 @@
+---
+type: docs
+title: "How-To: Query state"
+linkTitle: "How-To: Query state"
+weight: 250
+description: "API for querying state stores"
+---
+
+{{% alert title="alpha" color="warning" %}}
+The state query API is in **alpha** stage.
+{{% /alert %}}
+
+## Introduction
+
+The state query API provides a way of querying the key/value data stored in state store components. This query API is not a replacement for a complete query language, and is focused on retrieving, filtering and sorting key/value data that you have saved through the state management APIs.
+
+Even though the state store is a key/value store, the `value` might be a JSON document with its own hierarchy, keys, and values.
+The query API allows you to use those keys and values to retrive corresponding documents.
+
+This query API does not support querying of actor state stored in a state store. For that you need to use the query API for the specific database.
+See [querying actor state]({{< ref "state-management-overview.md#querying-actor-state" >}}).
+
+You can find additional information in the [resources]({{< ref "#resources" >}}) section.
```

Comment:
```suggestion
You can find additional information in the [Related links]({{< ref "#related-links" >}}) section.
```

Reply:
@dmitsh - Getting really close! Just a few more updates that I have suggestions for!